src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/OrdinaryMethodReferenceFinder.cs

```diff
@@ -121,7 +134,12 @@ private static bool IsGetAwaiterMethod(IMethodSymbol methodSymbol)
 
             var suppressionReferences = await FindReferencesInDocumentInsideGlobalSuppressionsAsync(
                 symbol, state, cancellationToken).ConfigureAwait(false);
-            return nameMatches.Concat(forEachMatches, deconstructMatches, getAwaiterMatches, suppressionReferences);
+
+            var addMatches = IsAddMethod(symbol)
```

Comment:
not entirely related to this PR, but do we already support object initializers?

Reply:
Yes we do.  Those are normal identifier references, so they fall out of normal searches.  collection-initializers are an interesting duck where the 'Add' methods are implicitly referenced and need to be specially found.